training/cifar/ds_config.json

```diff
@@ -23,7 +23,7 @@
   },
   "gradient_clipping": 1.0,
   "prescale_gradients": false,
-  "fp16": {
+  "bf16": {
```

Comment:
Support both fp16 and bf16.

Reply:
Done, now ds_config is defined in cifar10_deepspeed.py.  Both datatype and stage can be specified by command line argument.